### PR TITLE
支持返回 IPv6 订阅地址

### PR DESCRIPTION
**Pull request by JoeyLearnsToCode**

### CODE REVIEW

File: src/client/repo.ts [hunk 1]
```diff
@@ -152,6 +152,19 @@ export const getIPv4All = async (
     loss <= LOSS_THRESHOLD && delay <= DELAY_THRESHOLD)
 }
 
+function splitIpPort(address: string): [string, string] {
+  // 首先检查是否IPv6
```
Comment (yuchanns) [inline src/client/repo.ts:156]: Please use English

File: src/server/sub.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const sub = async (
   proxyFormat: ProxyFormat, isAndroid: boolean,
 ) => {
   // TODO: support IPv6
```
Comment (yuchanns) [inline src/server/sub.ts:16]: The comment should be removed

**Review by yuchanns — COMMENTED** [review]
Thanks

File: src/server/sub.ts (hunk 2 above)
Comment (JoeyLearnsToCode) [reply src/server/sub.ts]: done

**Pull request merged**